deepchecks/utils/performance/partition.py

```diff
@@ -22,27 +23,38 @@
 # TODO: move tabular functionality to the tabular sub-package
 
 
-__all__ = ['partition_column', 'DeepchecksFilter']
+__all__ = ['partition_column', 'DeepchecksFilter', 'convert_tree_leaves_into_filters', 'intersect_two_filters']
 
 
 class DeepchecksFilter:
     """Contains a filter function which works on a dataframe and a label describing the filter.
 
     Parameters
     ----------
-    filter_func : Callable
-        function which receive dataframe and return a filter on it
-    label : str
+    filter_functions : Callable, default: None
+        List of functions which receive dataframe and return a filter on it. If None uses applies no filter
+    label : str, default = ''
         name of the filter
     """
 
-    def __init__(self, filter_func: Callable, label: str):
-        self.filter_func = filter_func
+    def __init__(self, filter_functions: List[Callable] = None, label: str = ''):
+        if not filter_functions or len(filter_functions) == 0:
```

Comment:
if len(filter_functions) == 0 is True, then "not filter_functions" will be True as well so redundant